### ISSUE TITLE
Crash with `index out of range`

### ISSUE THREAD

**Issue by AltGr** [jech/galene#75]
We have the current crash of the server from time to time with 15+ users:
```
Apr 12 14:23:16 galene2 galene[57128]: panic: runtime error: index out of range [24] with length 24
Apr 12 14:23:16 galene2 galene[57128]: goroutine 150549 [running]:
Apr 12 14:23:16 galene2 galene[57128]: github.com/jech/galene/packetcache.(*Cache).GetAt(0xc000594b60, 0xc00018565c, 0xc0007ad800, 0x5e0, 0x5e0, 0x0)
Apr 12 14:23:16 galene2 galene[57128]:         /home/ubuntu/galene/packetcache/packetcache.go:418 +0x17a
Apr 12 14:23:16 galene2 galene[57128]: github.com/jech/galene/rtpconn.rtpWriterLoop(0xc00919af60, 0xc0000b4fd0, 0xc0011de6e0)
Apr 12 14:23:16 galene2 galene[57128]:         /home/ubuntu/galene/rtpconn/rtpwriter.go:323 +0xa0c
Apr 12 14:23:16 galene2 galene[57128]: created by github.com/jech/galene/rtpconn.newRtpWriter
Apr 12 14:23:16 galene2 galene[57128]:         /home/ubuntu/galene/rtpconn/rtpwriter.go:193 +0xf8
```
the number `24` appears to always be the same.
This is as of commit b660bc404cf2c933297501a2ab981070ea2faf28

**Comment by AltGr** [comment]
Should'nt it be the following ?
```patch
From 72d2cd293e6a0a52c1508be40b0b727c8bba2dd5 Mon Sep 17 00:00:00 2001
From: Louis Gesbert <louis.gesbert@ocamlpro.com>
Date: Mon, 12 Apr 2021 15:02:43 +0000
Subject: [PATCH] Fix index out of range in packetcache

---
 packetcache/packetcache.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packetcache/packetcache.go b/packetcache/packetcache.go
index d48eb33..ec974fb 100644
--- a/packetcache/packetcache.go
+++ b/packetcache/packetcache.go
@@ -412,7 +412,7 @@ func (cache *Cache) GetAt(seqno uint16, index uint16, result []byte) uint16 {
        cache.mu.Lock()
        defer cache.mu.Unlock()
 
-       if int(index) > len(cache.entries) {
+       if int(index) >= len(cache.entries) {
                return 0
        }
        if cache.entries[index].seqno != seqno {
-- 
2.27.0

```

**Comment by jech** [comment]
Fixed in 9533c208863cfefde2af5227219ead61db00d207, thanks.
